src/PharoThings-RemoteToolsServer.package/PotRemoteBoard.class/class/publishedService.st

```diff
@@ -0,0 +1,5 @@
+published service
+publishedService
+	<classAnnotation>
+
+	^TlpPublishedService named: PotBoard current displayString instance: PotBoard current
```

Comment:
It was not desired. It was from my branch in TelePharo with radar.
I will remove it